specification/resources/resource-manager/Microsoft.Solutions/stable/2017-09-01/managedapplications.json

```diff
@@ -1357,6 +1374,49 @@
           "type": "string"
         }
       }
+    },
+    "OperationDisplay": {
+      "properties": {
+        "provider": {
+          "type": "string",
+          "description": "Operation provider."
+        },
+        "resource": {
+          "type": "string",
+          "description": "Operation resource."
+        },
+        "operation": {
+          "type": "string",
+          "description": "Operation."
+        },
+        "description": {
+          "type": "string",
+          "description": "Operation description."
+        }
+      }
+    },
+    "OperationDefinition": {
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The name of the operation."
+        },
+        "display": {
+          "$ref": "#/definitions/OperationDisplay",
+          "description": "The operation display definition."
+        }
+      }
+    },
+    "OperationsList": {
+      "properties": {
+        "operations": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/OperationDefinition"
+          },
+          "description": "The array of solution operations."
+        }
```

Comment:
Add property since we recommended to apply  `x-ms-pageable` in the API

```
        "nextLink": {
          "readOnly": true,
          "type": "string",
          "description": "URL to get the next set of operation list results if there are any."
        }
```

see [this](https://github.com/Azure/azure-rest-api-specs/blob/1ed16067a3dd7534f40a2996b44a5a9a0dd782f2/specification/eventhub/resource-manager/Microsoft.EventHub/stable/2017-04-01/EventHub.json#L2376)